ujt/app.py

```diff
@@ -116,21 +136,21 @@ def generate_graph_elements(node_name_message_map: Dict[str, Node],
         A list of dictionary objects, each containing information regarding a single node (Service or Client) or edge (Dependency).
     """
 
-    compute_status.compute_node_statuses(node_name_message_map,
-                                         client_name_message_map)
-
     return (converters.cytoscape_elements_from_nodes(node_name_message_map) +
             converters.cytoscape_elements_from_clients(client_name_message_map))
 
 
-cyto.load_extra_layouts()
-app = dash.Dash(__name__)
-cache = Cache()
-cache.init_app(app.server,
-               config={
-                   "CACHE_TYPE": "filesystem",
-                   "CACHE_DIR": "cache_dir"
-               })
+@app.callback(
+    Output("cytoscape-graph", "elements"),
+    [Input("refresh-button", "n_clicks_timestamp")],
+)
+def refresh_graph(n_clicks_timestamp):
+    cache.clear()
```

Comment:
Force the cache to be cleared on button click. In the future, I will attach this callback to an [interval](https://dash.plotly.com/dash-core-components/interval) to refresh the graph automatically.